common/src/main/java/com/microsoft/identity/common/internal/ui/webview/OAuth2WebViewClient.java

```diff
@@ -136,18 +135,21 @@ public void onReceivedSslError(final WebView view,
         super.onReceivedSslError(view, handler, error);
         handler.cancel();
 
+        final String sslHelpUrl = "https://go.microsoft.com/fwlink/?linkid=2138180";
+        final String errMsg = String.format(
+                "Received SSL Error during request. Please see %s for more info.",
+                sslHelpUrl
+        );
```

Comment:
Can this whole thing just be a static constant?

Reply:
Also, using concatenation is far more efficient than String.format, which we should probably avoid whenever feasible.